easybuild/easyconfigs/s/Salmon/Salmon-0.12.0-foss-2018b.eb

```diff
@@ -0,0 +1,32 @@
+easyblock = 'CMakeMake'
+
+name = 'Salmon'
+version = '0.12.0'
+
+github_account = 'COMBINE-lab'
+homepage = 'https://github.com/%(github_account)s/%(namelower)s'
+description = """Salmon is a wicked-fast program to produce a highly-accurate,
+ transcript-level quantification estimates from RNA-seq data."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+toolchainopts = {'pic': True, 'openmp': True}
+
+source_urls = [GITHUB_LOWER_SOURCE]
+sources = ['v%(version)s.tar.gz']
+checksums = ['91ebd1efc5b0b4c12ec6babecf3c0b79f7102e42b8895ca07c8c8fea869fefa3']
+
+builddependencies = [('CMake', '3.12.1')]
+
+runtest = 'test'
+
+dependencies = [
+    ('Boost', '1.67.0'),
+    ('tbb', '2018_U5'),
```

Comment:
@timeu Why was `Jellyfish` dropped as a dependency here?

Reply:
@boegel Jellyfish isn't needed anymore since version 0.10.0: 

> The default k-mer class (used for certain operations within salmon) has been migrated from the jellyfish implementation to a custom implementation. This results in a small performance increase on our testing systems under linux, and a moderate performance increase under OSX.
>
>-- https://github.com/COMBINE-lab/salmon/releases/tag/v0.10.0

Reply:
OK, thanks for clarifying!